README.md

```diff
@@ -4,10 +4,36 @@
+| Deutsch-Jozsa | [Deutsch_Jozsa_Algorithm.ipynb](https://github.com/amazon-braket/amazon-braket-algorithm-library/blob/main/notebooks/textbook/Deutsch_Jozsa_Algorithm.ipynb) | [Deutsch1992](https://royalsocietypublishing.org/doi/10.1098/rspa.1992.0167) |
+| Grover's Search | [Grovers_Search.ipynb](https://github.com/amazon-braket/amazon-braket-algorithm-library/blob/main/notebooks/textbook/Grovers_Search.ipynb) | [Figgatt2017](https://www.nature.com/articles/s41467-017-01904-7), [Baker2019](https://arxiv.org/abs/1904.01671) |
+| QAOA | [Quantum_Approximate_Optimization_Algorithm.ipynb](https://github.com/amazon-braket/amazon-braket-algorithm-library/blob/main/notebooks/textbook/Quantum_Approximate_Optimization_Algorithm.ipynb) | [Farhi2014](https://arxiv.org/abs/1411.4028) |
+| Quantum Circuit Born Machine | [Quantum_Circuit_Born_Machine.ipynb](https://github.com/amazon-braket/amazon-braket-algorithm-library/blob/main/notebooks/textbook/Quantum_Circuit_Born_Machine.ipynb) | [Benedetti2019](https://www.nature.com/articles/s41534-019-0157-8),  [Liu2018](https://journals.aps.org/pra/abstract/10.1103/PhysRevA.98.062324) |
+| Quantum PCA | [Quantum_Principal_Component_Analysis.ipynb](https://github.com/amazon-braket/amazon-braket-algorithm-library/blob/main/notebooks/advanced_algorithms/Quantum_Principal_Component_Analysis.ipynb) | [He2022](https://ieeexplore.ieee.org/document/9669030) | 
+| QFT | [Quantum_Fourier_Transform.ipynb](https://github.com/amazon-braket/amazon-braket-algorithm-library/blob/main/notebooks/textbook/Quantum_Fourier_Transform.ipynb) | [Coppersmith2002](https://arxiv.org/abs/quant-ph/0201067) | 
+| QMC | [Quantum_Computing_Quantum_Monte_Carlo.ipynb](https://github.com/amazon-braket/amazon-braket-algorithm-library/blob/main/notebooks/advanced_algorithms/Quantum_Computing_Quantum_Monte_Carlo.ipynb) | [Motta2018](https://wires.onlinelibrary.wiley.com/doi/10.1002/wcms.1364), [Peruzzo2014](https://www.nature.com/articles/ncomms5213) |
+| QPE | [Quantum_Phase_Estimation_Algorithm.ipynb](https://github.com/amazon-braket/amazon-braket-algorithm-library/blob/main/notebooks/textbook/Quantum_Phase_Estimation_Algorithm.ipynb) | [Kitaev1995](https://arxiv.org/abs/quant-ph/9511026) |
+| Quantum Walk | [Quantum_Walk.ipynb](https://github.com/amazon-braket/amazon-braket-algorithm-library/blob/main/notebooks/textbook/Quantum_Walk.ipynb) | [Childs2002](https://arxiv.org/abs/quant-ph/0209131) |
+|Shor's| [Shors_Algorithm.ipynb](https://github.com/amazon-braket/amazon-braket-algorithm-library/blob/main/notebooks/textbook/Shors_Algorithm.ipynb) | [Shor1998](https://arxiv.org/abs/quant-ph/9508027) |
+| Simon's | [Simons_Algorithm.ipynb](https://github.com/amazon-braket/amazon-braket-algorithm-library/blob/main/notebooks/textbook/Simons_Algorithm.ipynb) | [Simon1997](https://epubs.siam.org/doi/10.1137/S0097539796298637) |
+
+---
+### Community repos
```

Comment:
I think we want a warning here like:

```
> :warning: **The following includes projects that are not provided by AWS. You are solely responsible for your use of those projects (including compliance with any applicable licenses and fitness of the project for your particular purpose).** :warning:
```

similar to the one at https://github.com/amazon-braket/amazon-braket-labs/blob/main/README.md.